tests/trainer/properties/test_log_dir.py

```diff
@@ -119,6 +119,7 @@ def test_logdir_logger_collection(tmpdir):
         max_steps=2,
         logger=[TensorBoardLogger(save_dir=save_dir, name="custom_logs")],
     )
+    # TODO: What behavior do we want once trainer.logger is deprecated?
```

Comment:
How should this unit test be handled? Part of this issue https://github.com/PyTorchLightning/pytorch-lightning/pull/11731/files#r801162619.